ui/src/app/cohort-search/search-bar/search-bar.component.ts

```diff
@@ -135,11 +137,11 @@ export class SearchBarComponent implements OnInit, OnDestroy {
     this.subscription.unsubscribe();
   }
 
-  inputChange(newVal: string) {
-    this.typedTerm = newVal;
+  inputChange() {
```

Comment:
Driveby comment: I happened to notice this behavior yesterday when I was doing the release QA. Maybe you're already planning this or its happening elsewhere, but I would recommend debouncing these queries i.e. if I quickly type in "heart", we ideally don't want to dispatch requests for "he", "hea", "hear", "heart". Debounce throttles inputs if they come in too quickly and ignores intermediates.

Example:
https://github.com/all-of-us/workbench/blob/fc988fc05b1e321b9fb0839abef3f041d9dc301d/public-ui/src/app/views/quick-search/quick-search.component.ts#L97

Reply:
Another example here: https://github.com/all-of-us/workbench/blob/master/ui/src/app/views/workspace-share/component.ts#L64

Reply:
Nice catch. I switched the input to use reactive forms and added debounce.